Yona/Yona.xcodeproj/xcshareddata/xcschemes/Yona.xcscheme

```diff
@@ -62,7 +62,7 @@
       </AdditionalOptions>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```

Comment:
So if I build for QA it will always build as Release? The issue was they were getting messages that show in DEBUG...